slotfills/sections/repeatable.tsx

```diff
@@ -11,21 +11,54 @@ import AddMoreButton from '../../components/add-more-button';
 
 import './repeatable.scss';
 
-interface ChildProps {
+const CustomKnob = React.forwardRef<HTMLDivElement, {}>((props, ref) => (
+  <span ref={ref} className="fm-gutenberg-move-handle" aria-label={__('Move', 'fm-gutenberg')}>::</span>
+));
+
+interface MaybeSortableListProps {
   children?: React.ReactNode | React.ReactNode[];
+  onSortEnd: (oldIndex: number, newIndex: number) => void;
+  sortable: boolean;
 }
 
-const SortableItem = SortableElement(({ children }: ChildProps) => (
-  <li>
-    {children}
-  </li>
-));
+function MaybeSortableList({ children, onSortEnd, sortable }: MaybeSortableListProps) {
+  return (
+    sortable ? (
+      <SortableList
+        onSortEnd={onSortEnd}
+        className="fm-gutenberg-sortable-list"
+      >
+        {children}
+      </SortableList>
+    ) : (
+      <ul
+        className="fm-gutenberg-list"
+      >
+        {children}
+      </ul>
+    )
+  );
+}
 
-const SortableList = SortableContainer(({ children }: ChildProps) => (
-  <ul className="fm-gutenberg-sortable-list">{children}</ul>
-));
+interface MaybeSortableItemProps {
+  children?: React.ReactNode | React.ReactNode[];
```

Comment:
Does `React.ReactNode` also include `React.ReactNode[]`? I think it does, but not 💯